src/core/sys/posix/stdio.d

```diff
@@ -332,6 +332,8 @@ else version (OpenBSD)                      // as of OpenBSD 5.4
     version = HaveMemstream;
 else version (CRuntime_UClibc)
     version = HaveMemstream;
+else version(CRuntime_Musl)
```

Comment:
```suggestion
else version (CRuntime_Musl)
```

Reply:
Ah thanks. Is there some standard formatter for the druntime? dfmt didn't like the alignment by variable name in this file.

Reply:
`make -f posix.mak style`

Reply:
Will do that next time, thanks :)